lib/components/tabs/tabs.dart

```diff
@@ -0,0 +1,136 @@
+import 'dart:html';
+import 'dart:async';
+import 'package:katikati_ui_lib/components/logger.dart';
+
+var logger = Logger('Tabs');
+
+class TabView {
+  String id;
+  String label;
+  DivElement content;
+
+  TabView(this.id, this.label, this.content);
+}
+
+class TabsView {
+  DivElement renderElement;
+  List<TabView> _tabs;
+  Map<String, TabView> _tabsLookup;
+
+  String _selectedTabID;
```

Comment:
nit: s/_selectedTabId/_selectedTabId
(the Dart style guide recommends that abbreviations (like ID) and acronyms with >=3 letters are capitalised like normal words)